assets/stylesheets/modules/_packages.scss

```diff
@@ -7,18 +7,18 @@
 
```

Comment:
Are you sure modifying this existing style won't break any other part of the site?